services/workflows-service/plugins/verify-repository-project-scoped/verify-repository-project-scoped.rule.js

```diff
@@ -17,7 +17,10 @@ module.exports = {
     return {
       MethodDefinition: function (node) {
         if (!isRepository || node.key.name === 'constructor') return;
-        const isUnscoped = node.key.name.toLowerCase().includes('unscoped');
+        const isUnscoped =
```

Comment:
Can this be an array of strings instead?

Reply:
@Blokh